package.json

```diff
@@ -29,7 +30,11 @@
     "eslint": "^8.46.0",
     "eslint-plugin-astro": "^0.27.2",
     "eslint-plugin-jsx-a11y": "^6.7.1",
+    "postcss-html": "^1.7.0",
     "prettier": "^3.0.1",
-    "prettier-plugin-astro": "^0.11.0"
+    "prettier-plugin-astro": "^0.11.0",
+    "stylelint": "^16.8.2",
+    "stylelint-config-html": "^1.1.0",
+    "stylelint-config-standard": "^36.0.1"
```

Comment:
curious: Which guide did you follow to install 5 packages for stylelint?

Reply:
[Astro doc](https://docs.astro.build/en/editor-setup/#stylelint) says can follow https://github.com/ota-meshi/stylelint-config-html
and `create-stylelint` just use for init the config